helloworld/test/greet-test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const chai = require('chai');
+const greet = require('../lib/greet.js');
+
+const expect = chai.expect;
+
+describe('testing module greet.js', function(){
+  describe('with valid inputs', function(){
+    it('should return "hello world"', function(){
+      expect(greet('world')).to.equal('hello world');
+    });
+
+    it('should return "hello bangarang"', function(){
+      expect(greet('bangarang')).to.equal('hello bangarang');
+    });
+  });
+
+  describe('with invalid inputs', function(){
```

Comment:
Remember again that the instructions needed some kind of handling to make sure the passed-in arg is a string, so your test should also be testing for that too.